src/pages/waiting-room-to-car/cat-c/components/vehicle-checks-modal/__tests__/vehicle-checks-modal.cat-c.analytics.effects.spec.ts

```diff
@@ -0,0 +1,151 @@
+import { TestBed } from '@angular/core/testing';
+import { VehicleChecksModalCatCAnalyticsEffects } from '../vehicle-checks-modal.cat-c.analytics.effects';
+import { Store, StoreModule } from '@ngrx/store';
+import { StoreModel } from '../../../../../../shared/models/store.model';
+import { ReplaySubject } from 'rxjs/ReplaySubject';
+import { testsReducer } from '../../../../../../modules/tests/tests.reducer';
+import { AnalyticsProvider } from '../../../../../../providers/analytics/analytics';
+import { AnalyticsProviderMock } from '../../../../../../providers/analytics/__mocks__/analytics.mock';
+import { provideMockActions } from '@ngrx/effects/testing';
+import * as testsActions from '../../../../../../modules/tests/tests.actions';
+import { TestCategory } from '@dvsa/mes-test-schema/categories/common/test-category';
+import { VehicleChecksViewDidEnter } from '../vehicle-checks-modal.cat-c.actions';
+import { AnalyticRecorded } from '../../../../../../providers/analytics/analytics.actions';
+import { AnalyticsEventCategories, AnalyticsScreenNames } from '../../../../../../providers/analytics/analytics.model';
+
+// TODO: MES-4254 import cat c actions
```

Comment:
When is this todo going to be implemented?

Reply:
So there is a sub-task on this story MES-4254 which is Integrate screens. There is a high chance that all (at least the majority) of these todos, marked with MES-4254 will be fixed as part of that subtask.

Reply:
Awesome, thanks Szabi.